src/Combinatorics/Matroids/properties.jl

```diff
@@ -1065,3 +1086,35 @@ function is_minor(Minor::Matroid, M::Matroid)
     end
     return false
 end
+
+@doc raw"""
+    matroid_base_polytope(M::Matroid) 
+
+The base polytope of the matroid `M`.  
+
+# Examples
+```jldoctest
+julia> D = matroid_base_polytope(uniform_matroid(2,4));
+
+julia> vertices(D)
+6-element SubObjectIterator{PointVector{QQFieldElem}}:
+ [1, 1, 0, 0]
+ [1, 0, 1, 0]
+ [1, 0, 0, 1]
+ [0, 1, 1, 0]
+ [0, 1, 0, 1]
+ [0, 0, 1, 1]
+```
+"""
+
+
+function indicator_vector(S::Vector{Int}, n::Int)
+    return map(x -> x in S ? 1 : 0 , 1:n)
+end
+
+function matroid_base_polytope(Q::Matroid)
+    n = length(matroid_groundset(Q))
+    Q = isomorphic_matroid(Q, [i for i in 1:n])
+    Delta_verts = hcat([indicator_vector(x, n) for x in bases(Q)]...)
+    return convex_hull(Delta_verts') 
+end
```

Comment:
````suggestion
@doc raw"""
    matroid_base_polytope(M::Matroid) 

The base polytope of the matroid `M`.  

# Examples
```jldoctest
julia> D = matroid_base_polytope(uniform_matroid(2,4));

julia> vertices(D)
6-element SubObjectIterator{PointVector{QQFieldElem}}:
 [1, 1, 0, 0]
 [1, 0, 1, 0]
 [1, 0, 0, 1]
 [0, 1, 1, 0]
 [0, 1, 0, 1]
 [0, 0, 1, 1]
```
"""
function matroid_base_polytope(Q::Matroid)
    n = length(matroid_groundset(Q))
    Q = isomorphic_matroid(Q, [i for i in 1:n])
    Delta_verts = hcat([indicator_vector(x, n) for x in bases(Q)]...)
    return convex_hull(Delta_verts') 
end


function indicator_vector(S::Vector{Int}, n::Int)
    return map(x -> x in S ? 1 : 0 , 1:n)
end
````
Really, the docstring is completely arbitrary to Julia. It will not be connected via some complicated parsing mechanism. It is only connected to the function that it is _directly above_.